pkgs/by-name/sq/sqlitestudio/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  libsForQt5,
+  readline,
+  tcl,
+  python3,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+stdenv.mkDerivation rec {
+  pname = "sqlitestudio";
+  version = "3.4.4";
+
+  src = fetchFromGitHub {
+    owner = "pawelsalawa";
+    repo = "sqlitestudio";
+    rev = version;
+    hash = "sha256-5oBYv8WxyfVvvqr15XApvn6P/lBxR8b6E+2acRkvX0U=";
+  };
+
+  nativeBuildInputs =
+    with libsForQt5.qt5;
+    [
+      qmake
+      qttools
+      wrapQtAppsHook
+    ]
+    ++ [ copyDesktopItems ];
+
+  buildInputs =
+    with libsForQt5.qt5;
+    [
+      qtbase
+      qtsvg
+      qtdeclarative
+      qtscript
+    ]
+    ++ [
+      readline
+      tcl
+      python3
+    ];
+
+  qmakeFlags = [ "../../SQLiteStudio3" ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      desktopName = "SQLiteStudio";
+      exec = pname;
+      icon = pname;
+      comment = "Database manager for SQLite";
+      terminal = false;
+      startupNotify = false;
+      categories = [ "Development" ];
+    })
+  ];
+
+  preConfigure = ''
+    mkdir -p output/build
+    cd output/build
+  '';
+
+  postInstall = ''
+    install -Dm755 \
+      ../../SQLiteStudio3/guiSQLiteStudio/img/sqlitestudio.svg \
+      $out/share/pixmaps/sqlitestudio.svg
+  '';
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    description = "A free, open source, multi-platform SQLite database manager";
+    homepage = "https://sqlitestudio.pl/";
+    license = licenses.gpl3;
+    mainProgram = pname;
```

Comment:
usage of pname is considered anti-pattern https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Thank you! I have removed all pname references.